openreview/venue_request/process/revisionProcess.py

```diff
@@ -14,8 +14,7 @@ def process(client, note, invitation):
 
     invitation_type = invitation.id.split('/')[-1]
     if invitation_type in ['Bid_Stage', 'Review_Stage', 'Meta_Review_Stage', 'Decision_Stage']:
-        if conference.submission_stage.double_blind and conference.submission_stage.due_date < datetime.datetime.now():
-            conference.create_blind_submissions()
+        conference.setup_venue_papers()
         conference.set_authors()
```

Comment:
is it possible to move these three lines to `setup_venue_papers`?

```
        conference.set_authors()
        conference.set_reviewers()
        if conference.use_area_chairs:
            conference.set_area_chairs()
```

It seems to be the same. We could rename the name of the function like `post_submission_stage`?

Reply:
Done. Method renamed to `setup_post_submission_stage`.